lib/tasks/migrate/20230730055732_migrate_from_folders_to_lists.rake

```diff
@@ -49,12 +49,17 @@ namespace :check do
       condition = {}
       if slugs.blank?
         last_team_id = Rails.cache.read('check:migrate:migrate_from_folders_to_lists:team_id') || 0
+        # Collect teams with activity in the last year
+        last_year = Time.now - 1.years
+        team_ids = ProjectMedia.select('team_id').where('updated_at > ?', last_year).group('team_id').map(&:team_id).uniq
```

Comment:
I have a question about this that I'm going to add to the ticket.

Reply:
changed condition to use `created_at` field